src/workerd/io/compatibility-date.capnp

```diff
@@ -947,4 +947,12 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # This flag is intended to be a roll-up flag. That is, as additional APIs
   # reach EOL, new compat flags will be added for those that will have
   # `impliedByAfterDate(name = "removeNodeJsCompatEOL", ...` annotations.
+
+  typescriptStripTypes @109 :Bool
+    $compatEnableFlag("typescript_strip_types")
+    $compatDisableFlag("no_typescript_strip_types")
```

Comment:
The expectation is that the flag will always be explicitly opt-in so a disable flag is unnecessary.
```suggestion
```